wolox-books/src/scss/colors.scss

```diff
@@ -3,4 +3,5 @@ $green: #BED23C;
 $lightgrey: #f1f1f1;
 $white: #FFF;
 $grey: #808080;
-$alto: #d3d3d3
+$alto: #d3d3d3;
+$black: #000000;
```

Comment:
when adding colors that look like #AABBCC write them as #ABC. `#000` for this particular case